tests/lang_callable/macros/tmodify_result_assignment.nim

```diff
@@ -3,7 +3,7 @@ discard """
     Partially typed ``nkReturnStmt`` AST is properly re-analyzed after macro
     invocation
   '''
-  errmsg: "got <string> but expected 'int'"
+  errormsg: "got <string> but expected 'int'"
   line: 26
```

Comment:
```suggestion
  line: 25
```

This will appease the tests, but I'm not sure if that's right given the `copyLineInfo` on line 22.

Reply:
The error should be on line 26, but the `copyLineInfo` copies to the wrong target node (`result.body` instead of `result.body[0][1]`).